### PR TITLE
Feat/189 support reco filters

### LINKED ISSUE #189: Support filters in Reco API
Support filters parameter in Reco API

## Expected Behaviour
Reco API should support `filters` parameter in the query and propagate it to the catalog-api

## Current Behaviour
Reco API doesn't support filtering

## Context
To show only zlabels brand products in the recommended carousel, we need to call the reco API with `type:cross_sell` and provide zlabels brands in `filters` parameter.

### LINKED ISSUE #190: Update Docs
Update docs, when a new version is published on npm

## Expected Behaviour
Whenever a new version is published to the npm registry, docs for the project should be updated which mentions the usage of the API, version etc.

## Current Behaviour
When a new version is published, docs are not updating

## Possible Solution
Update `.travis.yml` to run `npm run release:production` in the pipeline which will update new build and docs as well

### PR DESCRIPTION
**Pull request by hm6293**
closes #189 , closes #190

### CODE REVIEW

File: src/api/atlas_sdk_client.js [hunk 1]
```diff
@@ -468,15 +468,25 @@ class AtlasSDKClient {
    *    }
    * });
    */
-  getRecommendations(sku, options = {
-    reco_id: '',
-    tracking_string: '',
-    type: ''
-  }) {
+  getRecommendations(sku, options) {
```
Comment (Etchs) [inline src/api/atlas_sdk_client.js]: Could we update the documentation of the `options` input to this method to include `type` and  `filters` as well?

Comment (hm6293) [reply src/api/atlas_sdk_client.js]: Done

**Comment by hm6293** [comment]
👍

File: src/api/atlas_sdk_client.js (hunk 1 above)
Comment (Etchs) [reply src/api/atlas_sdk_client.js]: Thanks for adding the description for `filters`. FYI: I also added the docs for `type`

**Comment by Etchs** [comment]
👍

**Comment by hm6293** [comment]
👍

**Comment by hm6293** [comment]
👍

**Comment by Etchs** [comment]
👍

**Pull request merged**